R/main.R

```diff
@@ -11,6 +11,40 @@ check_drjacoby_loaded <- function() {
   message("drjacoby loaded successfully!")
 }
 
+#------------------------------------------------
+#' @title Define parameters dataframe
+#'
+#' @description Provides a convenient way of defining parameters in the format
+#'   required by \code{run_mcmc()}.
+#'
+#' @param ... define parameters one at a time, for each one defining four
+#'   elements: name, min, max, init.
+#'
+#' @export
+
+define_params <- function(...) {

```

Comment:
Added a test to check the output of this function against a basic dataframe produced through base R.

Reply:
In offline conversation decided to drop param_i in later PR and move to block method instead